lib/models.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+import statsmodels.api as sm
+
+# define the statistical model
+def linear_acceleration_model(df):
+    y = df['height']
+    X = np.c_[
+        df['year']-1970,
+        (df['year'] - 1970) * (df['year'] - 1970),
+        np.cos(2*np.pi*(df['year']-1970)/18.613),
+        np.sin(2*np.pi*(df['year']-1970)/18.613)
+    ]
+    month = np.mod(df['year'], 1) * 12.0
+    names = ['Constant', 'Trend', 'Acceleration', 'Nodal U', 'Nodal V']
+    X = sm.add_constant(X)
+    model = sm.OLS(y, X, missing='drop')
+    fit = model.fit()
+    return fit, names
+
+
+# define the statistical model
+def quadratic_model(df, with_wind=True):
+    """This model computes a parabolic linear fit. This corresponds to the hypothesis that sea-level is accelerating."""
+    y = df['height']
+    X = np.c_[
+        df['year']-1970,
```

Comment:
there is quite some repitition in these models? maybe a seperate function to do this part per model?

Reply:
I am in the process of restructuring this. For now I have to do it like this. I will update this in a next release.